app/views/widgets/onlineTestProgress.scala.html

```diff
@@ -174,7 +175,7 @@
                         <p class="med-btm-margin" id="etrayHeading"><i class="fa the-icon fa-minus"></i>E-tray </p>
                         <div id="etrayBlurb">
                             <p id="etrayDescription" class="">This section is timed, after starting the section you'll
-                                have 80 minutes to complete it.</p>
+                                have @{dashboardPage.phase2TimeToCompleteTest.getOrElse(80) } minutes to complete it.</p>
```

Comment:
Why is this constant also in the view? It should be in the back-end service, or at least passed from the controller